fusion-documentation/src/main/minisite/content/fusion/json.adoc

```diff
@@ -132,3 +132,48 @@ final var object = Map.of("foo", "bar");
 final var patched = patch.apply(object);
 // patched={"foo":"bar","baz":"qux"}
 ----
+
+== Json Schema Validator
+
+The Json Schema Validator can be used to validate a json content according to a json-schema (https://json-schema.org/understanding-json-schema/reference).
+
+[source,java]
+----
+@Bean
+public class MyService {
+
+    @Injection
+    JsonMapper mapper;
+
+    public boolean validate(final MyModel myModel) throws IOException {
+        final var schema = jsonMapper.fromString(Object.class,
+                """
+                        {
+                          "$id": "https://spec.openapis.org/oas/3.1/schema/2022-10-07",
+                          "$schema": "https://json-schema.org/draft/2020-12/schema",
+                          "description": "The description of OpenAPI v3.1.x documents without schema validation, as defined by https://spec.openapis.org/oas/v3.1.0",
+                          "type": "object"
+                          ...
+                        }
+                      """); <1>
+        final JsonSchemaValidatorFactory factory = new JsonSchemaValidatorFactory();
```

Comment:
maybe it is worth mentionning the newInstance result can be reused and is thread safe

Reply:
done